spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/source/SparkSQLExecutionHelper.java

```diff
@@ -53,7 +53,7 @@ public static String lastExecutedMetricValue(SparkSession spark, String metricNa
     // Refresh metricValues, they will remain null until the execution is complete and metrics are
     // aggregated
     Awaitility.await()
-        .atMost(Duration.ofMillis(500))
+        .atMost(Duration.ofSeconds(10))
```

Comment:
It started becoming flaky it seems. 
Ref: https://github.com/apache/iceberg/actions/runs/6678213413/job/18149016719

Reply:
this shouldn't be 10 seconds, but maybe rather 1-3 seconds. The old version of the code was relying on something around 500 millis